extensions/trace_propagators/src/main/java/io/opentelemetry/extensions/trace/propagation/Common.java

```diff
@@ -44,7 +44,7 @@ static SpanContext buildSpanContext(String traceId, String spanId, String sample
           traceFlags,
           TraceState.getDefault());
     } catch (Exception e) {
-      logger.log(Level.INFO, "Error parsing header. Returning INVALID span context.", e);
+      logger.log(Level.FINE, "Error parsing header. Returning INVALID span context.", e);
```

Comment:
This is mostly inconsistent, but we should probably use logger.info everywhere instead of logger.log(info,...)

Reply:
This is a bit, so no need to really change here, but maybe good to have consistent.